specutils/fitting/fitmodels.py

```diff
@@ -97,6 +97,7 @@ def _consecutive(data, stepsize=1):
     return np.split(data, np.where(np.diff(data) != stepsize)[0]+1)
 
 
+@warn_continuum_below_threshold(threshold=0.01, check=True)
```

Comment:
I think you don't want `check=True` here (or below), right?  Because then the `do_continuum_function_check` option gets ignored?

Reply:
hah, good point